README.md

```diff
@@ -84,9 +84,9 @@ Commitizen is great on its own, but it shines when you use it with some other am
 
 #### Retrying failed commits
 
-As of version 2.7.1, you may attempt to retry the last commit using the `git cz --retry` command.  This can be helpful when you have tests set up to run via a git precommit hook.  In this scenario, you may have attempted a Commitizen commit, painstakingly filled out all of the commitizen fields, but your tests fail. In previous Commitizen versions, after fixing your tests, you would be forced to fill out all of the fields again. Enter the retry command. Commitizen will retry the last commit that you attempted in this repo without you needing to fill out the fields again. 
+As of version 2.7.1, you may attempt to retry the last commit using the `git cz --retry` command or `npm run git-cz -- --retry` if running from a npm script.  This can be helpful when you have tests set up to run via a git precommit hook.  In this scenario, you may have attempted a Commitizen commit, painstakingly filled out all of the commitizen fields, but your tests fail. In previous Commitizen versions, after fixing your tests, you would be forced to fill out all of the fields again. Enter the retry command. Commitizen will retry the last commit that you attempted in this repo without you needing to fill out the fields again. 
```

Comment:
`npm run git-cz` wont work unless you have a `git-cz` script (often people will call the script `commit`). You can, however use https://www.npmjs.com/package/npm-run and do that :-) I think the wording here makes that a little unclear.

You might also mention https://www.npmjs.com/package/npm-quick-run which does not require `--` to pass args :-)